### PR TITLE
NEO-875 Enable basic touch support

### PR DESCRIPTION
**Pull request by jaller94**
`NeoBoard.touchSupport = true;`

[Peek 2024-09-18 18-29.webm](https://github.com/user-attachments/assets/3ef19289-69ba-449a-aa02-a1444e381dd2)

**:heavy_check_mark: Checklist**

- [ ] A changeset describing the change and affected packages ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#changelog-and-versioning)).
- [ ] Added or updated documentation.
- [ ] Tests for new functionality and regression tests for bug fixes.
- [ ] Screenshots or videos attached (for UI changes).
- [ ] All your commits have a `Signed-off-by` line in the message ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#dco)).

### CODE REVIEW

File: matrix-neoboard-widget/src/AppContainer.tsx [hunk 1]
```diff
@@ -57,9 +57,9 @@ export const AppContainer = ({
             <MuiWidgetApiProvider
               widgetApiPromise={widgetApiPromise}
               widgetRegistration={{
-                name: 'NeoBoard',
-                // "pad" suffix to get a custom icon
-                type: 'net.nordeck.whiteboard:pad',
+                name: 'NeoMap',
+                // "map" suffix to get a custom icon
+                type: 'net.nordeck.neomap:map',
```
Comment (MTRNord) [inline matrix-neoboard-widget/src/AppContainer.tsx:62]: Is this intentional? 🤔

Comment (jaller94) [reply matrix-neoboard-widget/src/AppContainer.tsx:62]: No

**Comment by mgcm** [comment]
I tested this on a real iPad device and here's a summary of before and after testing the PR:

**Before (without this PR)**

- Can’t easily select any shapes, only by touching the edges (and doesn’t always work)
- There is no way of adding arrows, lines and free hand drawing
- Drag to select doesn’t work
- Text input is not possible
- Tap to create is OK but drag to create doesn’t work

**With this PR:**

- Easy to select big shapes by only touching
- Resizing works OK but moving shapes around is bad, as it doesn’t follow the location of the finger, instead moving smaller deltas
- You can add arrows, shapes and free hand drawing but selecting them is very hard
- Drag to select works
- Text input is still not possible
- Tap to create and drag to create both work

So there is still work to be done here.

**Review by mgcm — CHANGES_REQUESTED** [review]
please check my previous comment.

File: matrix-neoboard-widget/src/AppContainer.tsx (hunk 1 above)
Comment (jaller94) [reply matrix-neoboard-widget/src/AppContainer.tsx]: Fixed in 176ef0729f9bc77d69f06606ba6847fe45648a83

File: matrix-neoboard-widget/src/AppContainer.tsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ export const AppContainer = ({
               widgetApiPromise={widgetApiPromise}
               widgetRegistration={{
                 name: 'NeoBoard',
-                // "pad" suffix to get a custom icon
+                // "map" suffix to get a custom icon
```
Comment (HarHarLinks) [inline matrix-neoboard-widget/src/AppContainer.tsx:61]: ?

Comment (jaller94) [reply matrix-neoboard-widget/src/AppContainer.tsx:61]:
```suggestion
                // "pad" suffix to get a custom icon
```

**Comment by jaller94** [comment]
Closed in favour of #614

**Pull request closed without merging**